examples/jsm/nodes/accessors/InstanceNode.js

```diff
@@ -8,7 +8,6 @@ import {
 	element,
 	dot,
 	div,
-	temp,
```

Comment:
I think it would be better hen to rename the 'label' to 'temp'.
I was imagining the label not only to variable but using in VarTemp too, but I think we can see that later.

Reply:
I think that `temp` in users' code can be used almost only as a cache-forcing mechanism (like in https://github.com/mrdoob/three.js/issues/23728, but it should be fixed now?), so in most cases `nodeProxy( VarNode )` would be used for labelling, so it would make more sense to name it `label`...

Reply:
I think that `temp` is a name more intuitive to force a variable creation, and the name can be optional.

Another usage for `temp( node, propertyName = null )` can be: 

```js
const vec = temp( vec3() );

vec.add( ...someNode );
```

Reply:
I do not think this usage should actually be - it is better to write `add( vec3(), ...someNodes )`.

Maybe it would be better to export `temp` as an alias of `label`, like I did with `element` and `arrayElement`?

Reply:
> I do not think this usage should actually be - it is better to write add( vec3(), ...someNodes ).

Maybe, but if the goal is to increment like a `inout/pointer` it should be similar to this `value.add( .. )` to preserve the reference.  For example:

```js
const computeMultiscattering = ( singleScatter, multiScatter, specularF90 = 1 ) => {

	const fab = DFGApprox.call( { roughness } );

	const FssEss = add( mul( specularColor, fab.x ), mul( specularF90, fab.y ) );

	const Ess = add( fab.x, fab.y );
	const Ems = sub( 1.0, Ess );

	const Favg = add( specularColor, mul( sub( 1.0, specularColor ), 0.047619 ) ); // 1/21
	const Fms = div( mul( FssEss, Favg ), sub( 1.0, mul( Ems, Favg ) ) );

	singleScatter.add( FssEss );
	multiScatter.add( mul( Fms, Ems ) );

};

const RE_IndirectSpecular_Physical = new ShaderNode( ( inputs ) => {

	const { radiance, irradiance, reflectedLight } = inputs;

	// Both indirect specular and indirect diffuse light accumulate here

	const singleScattering = temp( vec3() );
	const multiScattering = temp( vec3() );
	const cosineWeightedIrradiance = mul( irradiance, 1 / Math.PI );

	computeMultiscattering( singleScattering, multiScattering );

	const diffuse = mul( diffuseColor, sub( 1.0, add( singleScattering, multiScattering ) ) );

	reflectedLight.indirectSpecular.add( mul( radiance, singleScattering ) );
	reflectedLight.indirectSpecular.add( mul( multiScattering, cosineWeightedIrradiance ) );

	reflectedLight.indirectDiffuse.add( mul( diffuse, cosineWeightedIrradiance ) );

} );
```

Reply:
Fixed.